komodo-modeshape/pom.xml

```diff
@@ -23,5 +23,13 @@
 			</plugin>
 		</plugins>
 	</build>
+	
+	<dependencies>
+		<dependency>
+			<groupId>log4j</groupId>
+			<artifactId>log4j</artifactId>
+			<scope>compile</scope>
+		</dependency>
+	</dependencies>
```

Comment:
Can you provide a reason as to why this was required, please?

Reply:
This is paired with the other log4j change.  Since it is no longer a general compile time dependency, it needs to be added specifically for modeshape.